src/routes/Redeem.tsx

```diff
@@ -238,7 +310,23 @@ export function Redeem() {
                             </div>
                             {/* TODO: add payment details */}
                         </SuccessModal>
-                        <pre>NICE</pre>
```

Comment:
why delete this nice?

Reply:
idk what it's even doing there tbh, it should be behind the successmodal anyway